src/components/lesson/LessonsPane.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import LessonLink from './LessonLink';
```

Comment:
Import using the `index.js` file for consistency

```suggestion
import { LessonLink } from '..';
```